### ISSUE TITLE
Switch all `[DataTestMethod]` to `[TestMethod]`

### ISSUE THREAD

**Issue by Youssef1313** [microsoft/axe-windows#1074]
See https://github.com/microsoft/testfx/issues/4166

**Comment by codeofdusk** [comment]
[Sounds reasonable](https://stackoverflow.com/questions/46904953/datatestmethod-vs-testmethod). Are you willing to submit a PR?

**Comment by marcelwgn** [comment]
I think this issue has been addressed right?

**Comment by Youssef1313** [comment]
Yes. And I'm avoiding now the commented out occurrence in https://github.com/microsoft/axe-windows/pull/1093.

**Comment by codeofdusk** [comment]
We've typically kept issues open until the associated PR is included in a release. If and when we release next, this issue will be closed out.
